rasgoql/rasgoql/data/sqlalchemy.py

```diff
@@ -150,17 +61,11 @@ def change_namespace(self, namespace: str) -> None:
         `namespace`: str:
             namespace (database.schema)
         """
-        namespace = self.validate_namespace(namespace)
-        database, schema = self.parse_namespace(namespace)
-        try:
-            self.execute_query(f"USE DATABASE {database}")
-            self.execute_query(f"USE SCHEMA {schema}")
-            self.default_namespace = namespace
-            self.default_database = database
-            self.default_schema = schema
-            verbose_message(f"Namespace reset to {self.default_namespace}", logger)
-        except Exception as e:
-            self._error_handler(e)
```

Comment:
SQLAlchemy itself doesn't support changing DBs to my knowledge, and should create new engine instead. If we want to build a new engine here we could, but currently no subclasses do that.